Tests/Observable+MoyaSpec.swift

```diff
@@ -343,6 +343,108 @@ class ObservableMoyaSpec: QuickSpec {
                 expect(receivedObjects?.first?.createdAt) == formatter.date(from: "1995-01-14T12:34:56")!
             }
 
+            it("maps data representing an Int at a key path to an Int value") {
+                let json: [String: Any] = ["count": 1] // nested json array
+                guard let data = try? JSONSerialization.data(withJSONObject: json, options: .prettyPrinted) else {
+                    preconditionFailure("Failed creating Data from JSON dictionary")
+                }
+                let observable = Response(statusCode: 200, data: data).asObservable()
+
+                var count: Int?
+                _ = observable.map(Int.self, atKeyPath: "count", using: decoder).subscribe(onNext: { value in
+                    count = value
+                })
+                expect(count).notTo(beNil())
+                expect(count) == 1
+            }
+
+            it("maps data representing a Bool at a key path to a Bool value") {
```

Comment:
we should also add all of these to `Single+MoyaSpec` + `SignalProducer+MoyaSpec`